jdaviz/utils.py

```diff
@@ -355,6 +355,15 @@ def flux_conversion(spec, values, original_units, target_units):
     return (values * orig_units).to_value(targ_units, equivalencies=eqv)
 
 
+def _convert_surface_brightness_units(data, from_unit, to_unit):
+    try:
+        quantity = data * u.Unit(from_unit)
+        converted_quantity = quantity.to(u.Unit(to_unit))
+        return converted_quantity.value
```

Comment:
```suggestion
        converted_quantity = quantity.to_value(u.Unit(to_unit))
        return converted_quantity
```

Reply:
Do you not need the special equivalencies for this conversion?

Reply:
I think since this is surface brightness to surface brightness that we shouldn't need to use the scale factor here to convert

Reply:
No, as for the reasoning that Gilbert explained.